oremda/pipeline.py

```diff
@@ -222,3 +223,119 @@ def _terminate_operators(self, operators):
     def _create_queues(self, operators):
         with self.client.open_queue(OREMDA_FINISHED_QUEUE, create=True, reuse=True) as done_queue:
             pass
+
+def validate_port_type(type):
+    if type == PortType.Data:
+        return PortType.Data
+    elif type == PortType.Meta:
+        return PortType.Meta
+    else:
+        raise Exception(f'Unknown port type: {type}')
```

Comment:
A recommended (but not required) change for clarity:
```python
valid_types = [
    PortType.Data,
    PortType.Meta,
]

if type not in valid_types:
    raise Exception(f'Unknown port type: {type}')

return type
```

Having them in a list makes it easy to see which ones are valid, and also makes it easier to add/remove types from the list when maintaining it in the future.